src/commands/dev/index.js

```diff
@@ -577,7 +528,7 @@ DevCommand.flags = {
   live: flags.boolean({
     char: 'l',
     description: 'Start a public live session'
-  })
+  }),
```

Comment:
Shouldn't Prettier pick up those?

Reply:
I don't have prettier setup for `oncommit` hook, it messes up the staged hunks